templates/default/ndb_mgmd.service.erb

```diff
@@ -6,7 +6,7 @@ StartLimitBurst=20
 
 [Service]
 User = <%= node['ndb']['user'] %>
-PIDFile = <%= "#{node['ndb']['log_dir']}" %>/ndb_<%= @node_id %>.pid
+PIDFile = <%= "#{node['ndb']['log_dir']}" %>/<%= node['ndb']['mgmd']['pidfile'] %>
```

Comment:
we should update the pid in the stop script as well
https://github.com/logicalclocks/ndb-chef/blob/master/templates/default/mgm-server-stop.sh.erb#L13